### ISSUE TITLE
Fix welcome screen on small device with large accessibility text

### ISSUE THREAD

**Issue by alltheseas** [damus-io/damus#1608]
_what happens_
During onboarding, using a small screen with iOS accessibility font size turned up, I cannot create an account. The screen is cut off, and there is no scroll mechanism. 

https://damus.io/note1cgalvvhw6e6qcast586934328sd7kmdjsfy9z4tmx3fh7nlcxquqg8umea

![image](https://github.com/damus-io/damus/assets/64376233/b9527868-4b7a-42df-bc63-79ccca2d6368)

> I’m trying to download my friend iPhone for #Damus app. After I download the app, there’s no button to create an account or login option. Just like attached picture!!!

iPhone 8
